Stratis.Bitcoin.IntegrationTests/NodeBuilder.cs

```diff
@@ -355,7 +355,7 @@ public static NodeBuilder Create([CallerMemberNameAttribute] string caller = nul
             {
                 foreach (var bitcoind in Process.GetProcessesByName("bitcoind"))
                 {
-                    if (bitcoind.MainModule.FileName.Contains("Stratis.Bitcoin.Tests"))
+                    if (bitcoind.MainModule.FileName.Contains("Stratis.Bitcoin.IntegrationTests"))
                     {
                         bitcoind.Kill();
```

Comment:
How about using the stop rpc method? https://bitcoin.stackexchange.com/questions/13696/how-to-safely-shutdown-bitcoind

Reply:
`client.SendCommandAsync("stop");`

Reply:
That would be a nice improvement. This change simply fixes an oversight made when Neurosploit renamed the project on 3/7/2017. It is not attempting to change the method used at that time.